src/main/java/com/javadiscord/jdi/internal/models/auditlog/AuditLogEntry.java

```diff
@@ -0,0 +1,17 @@
+package com.javadiscord.jdi.internal.models.auditlog;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+// missing auditLogChanges
+@JsonIgnoreProperties(ignoreUnknown = true)
+public record AuditLogEntry(
+    @JsonProperty("target_id") String targetId,
```

Comment:
All IDs should be strings, can you confirm and make the changes across the models you have changed? I can do this in a seperate PR too so LMK

Reply:
Fixed